docs/search_dsl.rst

```diff
@@ -414,6 +413,30 @@ endpoint) you can do so via ``execute_suggest``:
 
     print(suggestions.my_suggestion)
 
+
+More Like This Query
+~~~~~~~~~~~~~~~~~~~~
+
+To use Elasticsearch's more_like_this functionality, you can use the MoreLikeThis query type.
+
+A simple example is below
+
+.. code:: python
+    from elasticsearch_dsl.query import MoreLikeThis
+    from elasticsearch_dsl Search
+
+    my_text = 'I want to find something similar'
+
+    s = Search()
+    # We're going to match based only on two fields, in this case text and title
+    s = s.query(MoreLikeThis(like=my_text, fields=['text', 'title]))
+    # You can also exclude fields from the result to make the response quicker in the normal way
+    s = s.source(exclude=["text"])
+    response = s.execute().to_dict()
```

Comment:
Hey @HonzaKral , sorry for coming out of the blue. I am executing this query on elasticsearch 5.5  but doesn't return any results, but returns when I do a norma multi_match search. Is this supported by all versions? 

Here is a snippet from my code: 
` 
  more_like_this = 'Something awesome'
  search = ProductDoc.search()
  query = MoreLikeThis(like=more_like_this, fields=['title', 'description', 'tags', 'categories'])
  search = search.query(query)
  return search.execute().to_dict()
`
Am I doing something wrong? Also tried the code above.

Also @eamonnmag  the second import looks wrong  maybe change it from
`from elasticsearch_dsl Search`
to 
`from elasticsearch_dsl import Search` ?